libnetdata/socket/socket.c

```diff
@@ -995,21 +995,89 @@ int accept4(int sock, struct sockaddr *addr, socklen_t *addrlen, int flags) {
 }
 #endif
 
+/*
+ * ---------------------------------------------------------------------------------------------------------------------
+ * connection_allowed() - if there is an access list then check the connection matches a pattern.
```

Comment:
Hi @amoss ,

Few days ago we agreed in Netdata to use `Doxygen` format to comment the functions, please, update this comment for their format.

Reply:
I was asked earlier in the thread to ignore the existing style and follow your style guide. It currently says that comments should be in the linux kernel style (which is how the comment is styled).